tests/testthat/test-syntactic-names.R

```diff
@@ -46,36 +46,36 @@ test_that("make_syntactic(): number", {
 
 test_that("make_syntactic(): number then character", {
   expect_syntactic(
-    c(  "0a",   "1b",   "22c",   "333d"),
-    c("..0a", "..1b", "..22c", "..333d")
+    c(   "0a",    "1b",    "22c",    "333d"),
+    c("...0a", "...1b", "...22c", "...333d")
```

Comment:
I commented in the code above, asking if `make_syntactic()` should be affected by this PR. If not, then the changes to these tests (and a few below) would be reversed.